anchorecli/cli/utils.py

```diff
@@ -37,10 +38,18 @@ def setup_config(cli_opts):
 
     settings = {}
 
+    # load environment if present
+    for e in ['ANCHORE_CLI_USER', 'ANCHORE_CLI_PASS', 'ANCHORE_CLI_URL', 'ANCHORE_CLI_HUB_URL', 'ANCHORE_CLI_API_VERSION', 'ANCHORE_CLI_SSL_VERIFY', 'ANCHORE_CLI_JSON', 'ANCHORE_CLI_DEBUG', 'ANCHORE_CLI_ACCOUNT', 'ANCHORE_CLI_CONFIG']:
+        if e in os.environ:
+            settings[e] = os.environ[e]
+
     # load up credentials file if present
     try:
-        home = os.path.expanduser('~')
-        credential_file = os.path.join(home, '.anchore', 'credentials.yaml')
+        if settings['ANCHORE_CLI_CONFIG']:
```

Comment:
This is causing a KeyError when not set. Try something like:
```suggestion
        if 'ANCHORE_CLI_CONFIG' in settings:
```